src/test/java/com/mate/fruitshop/dao/impl/FruitStorageDaoImplTest.java

```diff
@@ -0,0 +1,54 @@
+package com.mate.fruitshop.dao.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import com.mate.fruitshop.dao.FruitStorageDao;
+import com.mate.fruitshop.db.Storage;
+import com.mate.fruitshop.model.FruitEntry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitStorageDaoImplTest {
+    private static FruitStorageDao dao = new FruitStorageDaoImpl();

```

Comment:
dao is too incomprehensible name for object, i think fruitStorage or storage will be better

Reply:
DAO is not the same thing as storage, it is a class that accesses Storage - Database Access Object